spec/std/enumerable_spec.cr

```diff
@@ -747,12 +774,32 @@ describe "Enumerable" do
     it "rejects the values for which the block returns true" do
       [1, 2, 3, 4].reject(&.even?).should eq([1, 3])
     end
+
+    it "rejects with pattern" do
+      [1, 2, 3, 4].reject(&.even?).should eq([1, 3])
```

Comment:
You forgot to modify this example :)

Reply:
Wow, good catch! It seems I even forgot to implement `reject(pattern)`, lol :-P